tool.rkt

```diff
@@ -32,8 +33,8 @@ The maximize button of the frame also disappears, as if the X11 maximize propert
 
 (define orig-display-handler #f) ; will be set in the unit.
 
-(define (user-script-files #:exclude? [exclude? #t])
-  (lib:all-files (lib:load library-file) #:exclude? exclude?))
+(define (user-script-files)
+  (lib:all-enabled-scripts (lib:load)))
```

Comment:
I suspect it might be better for `(compile-library)` not to attempt to recompile scripts from collections, but maybe it's harmless?

Reply:
Matthew recently had some issues with this actually (personal email from Dec 15, 2023, cc robby): when working on the GUI, launching DrRacket was getting slow because some script compilation triggered compilation of the GUI
@rfindler said he'd think about it. Discussion still ongoing.